### ISSUE TITLE
Categorization of passwords

### ISSUE THREAD

**Issue by Megha66** [Pritom14/Password-Storage#38]
The app can sort passwords that have been stored in it. For eg. passwords related to office,home,social networking etc.

**Comment by Pritom14** [comment]
How do you want to proceed?

**Comment by Megha66** [comment]
I just thought of this idea.. Give me a few days to figure out what can be
done and how exactly do we have to proceed

**Comment by Pritom14** [comment]
Sure go ahead

**Comment by dastamoghno** [comment]
I also want to work on this issue.

**Comment by dastamoghno** [comment]
@Pritom14 do I need to learn firebase to implement this?

**Comment by yog-singh** [comment]
Tags can be added for each entry and then sorted using it.

**Comment by Pritom14** [comment]
@Megha66 , @dastamoghno , @yog-singh suggest your ideas to sort out this issue

**Comment by Pritom14** [comment]
Anybody working on this issue? If not I am going to close this

**Comment by yog-singh** [comment]
Yeah I'm working on the issues. Will soon update you with the approach I'm using.

**Comment by lukiee555** [comment]
@yog-singh you stilk working on this?

**Comment by Pritom14** [comment]
@lukiee555 , you can take up this issue, as because I haven't got any response from others

**Comment by Pritom14** [comment]
Also, how do you want to implement the issue?

**Comment by lukiee555** [comment]
as mention above I will categorized them eg. home, office etc
